docs/drivers.md

```diff
@@ -39,6 +40,19 @@ $ newgrp libvirtd
 $ newgrp libvirt
 ```
 
+#### Hyperkit driver
+
+The Hyperkit driver will eventually replace the existing xhyve driver.
+It is built from the minikube source tree, and uses [moby/hyperkit](http://github.com/moby/hyperkit) as a Go library.
+
+To install the hyperkit driver:
+
+```
+curl -Lo https://storage.googleapis.com/releases/latest/docker-machine-driver-xhyve && chmod +x docker-machine-driver-xhyve && sudo mv docker-machine-driver-xhyve /usr/local/bin/ && sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve && sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
```

Comment:
I don't think this is currently populated since I added it after the last release.  We might need to upload one manually in the meantime.